### ISSUE TITLE
IO error reading item overrides

### ISSUE THREAD

**Issue by TheDGOfficial** [emilyalexandra/chime#5]
Hi, I'm getting these when the game is loading or reloading resources.

```
[chime] IO error reading item overrides: generation marker
net.minecraft.class_151: Non [a-z0-9/._-] character in path of location: minecraft:generation marker
	at net.minecraft.class_2960.<init>(class_2960.java:41)
	at net.minecraft.class_2960.<init>(class_2960.java:46)
	at net.minecraft.class_793.calculateChimeOverrides(class_793.java:572)
	at net.minecraft.class_793.handler$zpl000$getModelDependencies(class_793.java:553)
	at net.minecraft.class_793.method_4755(class_793.java)
	at net.minecraft.class_1088.method_4729(class_1088.java:454)
	at net.minecraft.class_1088.method_4715(class_1088.java:337)
	at net.minecraft.class_1088.method_4726(class_1088.java:319)
	at net.minecraft.class_1088.method_4727(class_1088.java:458)
	at net.minecraft.class_1088.<init>(class_1088.java:206)
	at net.minecraft.class_1092.method_18178(class_1092.java:52)
	at net.minecraft.class_1092.method_18789(class_1092.java:19)
	at net.minecraft.class_4080.method_18791(class_4080.java:11)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1764)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1756)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)

[chime] IO error reading item overrides: block entity marker
net.minecraft.class_151: Non [a-z0-9/._-] character in path of location: minecraft:block entity marker
	at net.minecraft.class_2960.<init>(class_2960.java:41)
	at net.minecraft.class_2960.<init>(class_2960.java:46)
	at net.minecraft.class_793.calculateChimeOverrides(class_793.java:572)
	at net.minecraft.class_793.handler$zpl000$getModelDependencies(class_793.java:553)
	at net.minecraft.class_793.method_4755(class_793.java)
	at net.minecraft.class_1088.method_4729(class_1088.java:454)
	at net.minecraft.class_1088.method_4715(class_1088.java:337)
	at net.minecraft.class_1088.method_4726(class_1088.java:319)
	at net.minecraft.class_1088.method_4727(class_1088.java:458)
	at net.minecraft.class_1088.<init>(class_1088.java:206)
	at net.minecraft.class_1092.method_18178(class_1092.java:52)
	at net.minecraft.class_1092.method_18789(class_1092.java:19)
	at net.minecraft.class_4080.method_18791(class_4080.java:11)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1764)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1756)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by LambdAurora** [comment]
This is not a Chime issue but an issue with the resource pack you're using which is using invalid identifiers (have spaces in the path which is forbidden in Vanilla) `minecraft:block entity marker` and `minecraft:generation marker`.

The fix is to replace those spaces with underscores `_`.

**Comment by emilyalexandra** [comment]
Actually it's a chime issue, it doesn't cause any problems but it does junk up the log and will be fixed in the next release

**Comment by emilyalexandra** [comment]
Fixed in 1.1.3
